src/index.html

```diff
@@ -2,11 +2,348 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta 
+      name="viewport" 
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,900&display=swap" rel="stylesheet"
+    >
+    
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="moyo-logo">
+      </a> 
+      <nav class="nav">
```

Comment:
Add empty lines between multiline sibling blocks of HTML. Check it everywhere 
```suggestion
      </a> 
      
      <nav class="nav">
```